snapshots/tuberculosis/2023-11-27/data_dictionary.py

```diff
@@ -0,0 +1,24 @@
+"""Script to create a snapshot of dataset."""
```

Comment:
Ah good point, there are some other datasets I need to import that also use this dictionary, so figured it might be useful to have it as its own dataset. I will add a note that this is what I am doing, it does look a bit weird at the moment.